specifyweb/frontend/js_src/lib/components/Attachments/Viewer.tsx

```diff
@@ -93,49 +100,59 @@ export function AttachmentViewer({
   const table = f.maybe(serialized.tableID ?? undefined, getAttachmentTable);
   return (
-    <div className="flex h-full gap-8">
-      <div className="flex min-h-[30vw] w-full min-w-[30vh] flex-1 items-center">
-        {originalUrl === undefined ? (
-          loadingGif
-        ) : type === 'image' ? (
-          <img alt={title} src={originalUrl} />
-        ) : type === 'video' ? (
-          /*
-           * Subtitles for attachments not yet supported
-           */
-          // eslint-disable-next-line jsx-a11y/media-has-caption
-          <video
-            autoPlay={autoPlay}
-            className="h-full w-full"
-            controls
-            src={originalUrl}
-          />
-        ) : type === 'audio' ? (
-          /*
-           * Subtitles for attachments not yet supported
-           */
-          // eslint-disable-next-line jsx-a11y/media-has-caption
-          <audio
-            autoPlay={autoPlay}
-            className="w-full"
-            controls
-            src={originalUrl}
-          />
-        ) : (
-          <object
-            aria-label={title}
-            className="h-full w-full border-0"
-            data={originalUrl}
-            type={mimeType}
-          >
-            <img
-              alt={title}
-              className="h-full w-full object-scale-down"
-              src={thumbnail?.src}
+    <div className="flex h-full justify-center gap-8">
+      {displayOriginal === 'full' ? (
+        <div className="flex min-h-[30vw] w-full min-w-[30vh] flex-1 items-center">
+          {originalUrl === undefined ? (
+            loadingGif
+          ) : type === 'image' ? (
+            <img alt={title} src={originalUrl} />
+          ) : type === 'video' ? (
+            /*
+             * Subtitles for attachments not yet supported
+             */
+            // eslint-disable-next-line jsx-a11y/media-has-caption
+            <video
+              autoPlay={autoPlay}
+              className="h-full w-full"
+              controls
+              src={originalUrl}
+            />
+          ) : type === 'audio' ? (
+            /*
+             * Subtitles for attachments not yet supported
+             */
+            // eslint-disable-next-line jsx-a11y/media-has-caption
+            <audio
+              autoPlay={autoPlay}
+              className="w-full"
+              controls
+              src={originalUrl}
             />
-          </object>
-        )}
-      </div>
+          ) : (
+            <object
+              aria-label={title}
+              className="h-full w-full border-0"
+              data={originalUrl}
+              type={mimeType}
+            >
+              <img
+                alt={title}
+                className="h-full w-full object-scale-down"
+                src={thumbnail?.src}
+              />
+            </object>
+          )}
+        </div>
+      ) : (
+        <div className="flex items-center justify-center">
```

Comment:
can you refactor this file to use the same `<div>` for both cases?

something like this:
```tsx
<div className="flex min-h-[30vw] w-full min-w-[30vh] flex-1 items-center">
      {displayOriginal === 'full' ?
         originalUrl === undefined ? (
            loadingGif
          ) : type === 'image' ? (
            <img alt={title} src={originalUrl} />
          )
         // ...
        : <Thumbnail
            attachment={serializeResource(attachment)}
            className="!border-none"
            thumbnail={thumbnail}
          />
</div>
```

Reply:
Yes because I forgot to push it :D